README.md

```diff
@@ -2,10 +2,20 @@
 
 ScalarDL is a scalable and practical Byzantine fault detection middleware for transactional database systems that achieves correctness, scalability, and database agnosticism.
 
-## Docs
-* [ScalarDL Documentation](https://scalardl.scalar-labs.com/docs/latest/)
-* [ScalarDL Technical Overview](https://speakerdeck.com/scalar/scalar-dl-technical-overview)
-* [ScalarDL Research Paper](https://dl.acm.org/doi/abs/10.14778/3523210.3523212) [VLDB'22]
+To learn more, see [ScalarDL Overview](https://scalardl.scalar-labs.com/docs/latest/overview).
+
+## Quickstart
+
+1. Start running contracts.
+   - **Get started with ScalarDL Ledger:** To set up a basic application and run contracts by using ScalarDL Ledger, see [Get Started with ScalarDL Ledger](https://scalardl.scalar-labs.com/docs/latest/getting-started).
+   - **Get started with ScalarDL Auditor (for enterprise customers):** To set up a basic application and run contracts by using ScalarDL Ledger and then use ScalarDL Auditor to detect whether those contracts have been tampered with, see [Getting Started with ScalarDL Auditor](https://scalardl.scalar-labs.com/docs/latest/getting-started-auditor).
+2. See how to set up [a simple bank account application that uses ScalarDL](https://scalardl.scalar-labs.com/docs/latest/applications/simple-bank-account).
```

Comment:
Might be nice to mention it's a sample application like something following.

```suggestion
2. See how to set up [a simple bank account application that uses ScalarDL](https://scalardl.scalar-labs.com/docs/latest/applications/simple-bank-account), as a sample application.
```

Reply:
Thank you for the suggestion! I'm OK with mentioning that this one is a sample application👍